src/Qir/Controller/Constant.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Qir
+{
+    public static class Constant
```

Comment:
I would advocate for a `src` and a `bin` directory. So I think it would make sense to have a `DirectoryPath` static class that elements in the `FilePath` static class use.

Related to this [comment](https://github.com/microsoft/qsharp-runtime/pull/620/files#r610220933), the `DirectoryPath` static class would also be leveraged in that case.

Reply:
Makes sense. I'll address this in a future PR and we can discuss a little more offline.